parsons/ngpvan/bulk_import.py

```diff
@@ -153,3 +197,165 @@ def bulk_apply_activist_codes(self, tbl, url_type, **url_kwargs):
                                      [{"name": "ActivistCode"}],
                                      'Activist Code Upload',
                                      **url_kwargs)
+
+    def bulk_upsert_contacts(self, tbl, url_type, result_fields=None, **url_kwargs):
+        """
+        Bulk create or update contact records. Provide a Parsons table of contact data to
+        create or update records.
+
+        .. note::
+            * The first column of the table must be VANID.
```

Comment:
In the name of making this more user-friendly, could we move the VAN ID column to the beginning of the `Table` for the user?